cypress/integration/login.js

```diff
@@ -12,16 +12,17 @@ describe('Sourcebooks login', function() {
 
         cy.visit('/');
         cy.get('[id="loginForm.userId"]').click({force:true});
-        cy.get('[aria-label="Demo User"]').click();
+        cy.get('[aria-label="Ieva Stakneviciute"]').click();
         cy.get('[id="loginForm.role"]').click({force:true});
-        cy.get('[aria-label="User"]').click();
+        cy.get('[aria-label="Team Lead"]').click();
         cy.get('[type="submit"]').click();
 
         cy.url().should('include', '/time-logging');
         cy.get('.page__title').contains('Timesheets')
         cy.get('.calendar').should('be.visible')
         cy.get('.tile.form').should('be.visible')
-        cy.get('.user-info__title').contains('Demo User');
-        cy.get('.main-nav').find('li').should('have.length', 1);
+        cy.get('.user-info__title').contains('Ieva Stakneviciute');
+        cy.get('.main-nav').find('li').should('have.length', 2);
+        cy.get('.calendar__day.calendar--today.calendar--selected').contains('24');

```

Comment:
This locator is too long, it can be much shorter to find specific element. Also because this locator contains ".calendar--selected" it might be that if you select another date in the calendar it will fail to assert today's date